intermediate_source/tensorboard_profiler_tutorial.py

```diff
@@ -0,0 +1,240 @@
+"""
+PyTorch Tensorboard Profiler
+====================================
+This recipe explains how to use PyTorch tensorboard profiler
+and measure the performance bottleneck of the model.
+
+.. note::
+    PyTorch 1.8 introduces the new API that will replace the older profiler API
+    in the future releases. Check the new API at `this page <https://pytorch.org/docs/master/profiler.html>`__.
+
+Introduction
+------------
+PyTorch 1.8 includes an updated profiler API that could help user
+record both the operators running on host side and the CUDA kernels running on GPU side.
+It also includes a plugin that could show the performance bottleneck in visualization UI.
```

Comment:
The plugin is not included in pytorch. Could we  say it like: To visualize the profiling information generated by the profiler API, we also introduce a TensorBoard Plugin that ...

Reply:
Thanks, done.